lsp/package.json

```diff
@@ -15,7 +15,7 @@
         "format": "prettier --write .",
         "format-staged": "npx pretty-quick --staged",
         "clean": "ts-node ./script/clean.ts",
-        "compile": "tsc --build --verbose",
+        "compile": "tsc --build --verbose && npm run postcompile --workspaces --if-present",
```

Comment:
does npm implicitly run `postcompile` after `compile` https://docs.npmjs.com/cli/v9/using-npm/scripts#pre--post-scripts ? If so, it would seem redundant to call `postcompile` from `compile`.